src/main/java/de/zalando/aruha/nakadi/service/EventPublisher.java

```diff
@@ -65,6 +74,20 @@ public EventPublishResult publish(final JSONArray events, final String eventType
         }
     }
 
+    private void enrich(final List<BatchItem> batch, final EventType eventType) throws EnrichmentException {
+        for (BatchItem item : batch) {
+            try {
+                item.setStep(EventPublishingStep.ENRICHING);
+                enrichment.enrich(item.getEvent(), eventType);
```

Comment:
Actually there's no need to return the result since this method just updates the object. Do you think it would be better to change the interface not to return anything?